src/js/markup/create-products-list.ts

```diff
@@ -16,6 +15,51 @@ export function createProductsList(productsList: DataType[], ids: number[], urlA
           ${createSortOptions(filtredData.length)}
           ${filtredData.length && filtredData.length > DEFAULT_AMOUNT_ITEMS ? createPagination(urlPageNumber, urlAmountOfItems, amountPages) : ''}
          <ul class="products">
-          ${renderedData.length ? renderedData.map((product) => createItem(product, ids)).join('') : EMPTY_LIST}
+          ${renderedData.length ? renderedData.map((product) => createCatalogItem(product, ids)).join('') : EMPTY_LIST}
          </ul></section>`;
 }
+
+function createCatalogItem(product: DataType, ids: number[]) {
+  const { thumbnail, title, id, category, brand, price, discountPercentage, rating, stock } = product;
+  const isInCart = ids.includes(+id);
+
+  return `<li class="products__item products-item">
+            <span class="products-item__description">${title}</span>
+              <ul class="products-item__full-description full-description">
+              <li class="full-description__item">
+                <span>Category</span>
+                <span>${category}</span>
+              </li>
+              <li class="full-description__item">
+                <span>Brand</span>
+                <span>${brand}</span>
+              </li>
+              <li class="full-description__item">
+                <span>Price</span>
+                <span>$${price}.00</span>
+              </li>
+              <li class="full-description__item">
+                <span>Discount</span>
+                <span>%${discountPercentage}</span>
+              </li>
+              <li class="full-description__item">
+                <span>Rating</span>
+                <span>${rating}</span>
+              </li>
+              <li class="full-description__item">
+                <span>Stock</span>
+                <span>${stock}</span>
+              </li>
+            </ul>
+            <img

```

Comment:
img has attribute 'onerror' - it similar to click , but you can set another src in error case( if ref not work)